src/suricata-common.h

```diff
@@ -176,6 +180,10 @@
 #include <pcap/bpf.h>
 #endif
 
+#ifdef HAVE_SYS_TREE_H
+#include <bsd/sys/tree.h>
```

Comment:
Nope, these shouldn't be needed so we can remove libbsd all around.  I'll do that with an update.